### ISSUE TITLE
v1.0.2: Producing Infinite Newlines

### ISSUE THREAD

**Issue by jaswrks** [hosseinmd/prettier-plugin-jsdoc#204]
This behavior started in v1.0.2.

```
/**
 * A description.
 *
 * @note A note.
 *
 *
 *
 *
 *
 *
 *
 *
 *
 *
 *
 *
 *
 *
 *
 *
 *
 *
 *
 *
 *
 *
 * @see http://acme.com
 */
```

Each time I reformat, new lines are added before the `@see` line.

Prettier config values:

```
arrowParens: 'always',
bracketSameLine: false,
bracketSpacing: true,
embeddedLanguageFormatting: 'auto',
endOfLine: 'lf',
htmlWhitespaceSensitivity: 'css',
insertPragma: false,
jsxSingleQuote: true,
printWidth: 180,
proseWrap: 'preserve',
quoteProps: 'preserve',
requirePragma: false,
semi: true,
singleAttributePerLine: false,
singleQuote: true,
tabWidth: 4,
trailingComma: 'all',
useTabs: true,
vueIndentScriptAndStyle: true,

...

jsdocAddDefaultToDescription: false,
jsdocCapitalizeDescription: true,
jsdocDescriptionTag: false,
jsdocDescriptionWithDot: true,
jsdocKeepUnParseAbleExampleIndent: false,
jsdocLineWrappingStyle: 'greedy',
jsdocPreferCodeFences: false,
jsdocPrintWidth: 120,
jsdocSeparateReturnsFromParam: false,
jsdocSeparateTagGroups: true,
jsdocSingleLineComment: false,
jsdocSpaces: 1,
jsdocVerticalAlignment: true,
tsdoc: false,
```

**Comment by hosseinmd** [comment]
fixed 1.0.4

**Comment by jaswrks** [comment]
Thank you!
